code/__DEFINES/traits.dm

```diff
@@ -116,6 +116,8 @@
 /// Returns a list of trait sources for this trait. Only useful for wacko cases and internal futzing
 /// You should not be using this
 #define GET_TRAIT_SOURCES(target, trait) target._status_traits?[trait] || list()
+/// A simple helper for checking traits in a mob's mind
+#define HAS_MIND_TRAIT(target, trait) (HAS_TRAIT(target, trait) || (target.mind ? HAS_TRAIT(target.mind, trait) : null))
```

Comment:
```suggestion
#define HAS_MIND_TRAIT(target, trait) (HAS_TRAIT(target, trait) || (target.mind ? HAS_TRAIT(target.mind, trait) : FALSE))
```

c'mon nephew why are we involving `null` for a boolean sounding define that is always treated like a boolean? just use `FALSE` and keep our hands clean since this could be how databases get absolutely shittered a-la-#74856